std-bits/table/src/main/java/org/enso/table/excel/ExcelUtils.java

```diff
@@ -0,0 +1,71 @@
+package org.enso.table.excel;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.Temporal;
+
+public class ExcelUtils {
+  private static final LocalDate EPOCH_1900 = LocalDate.of(1899, 12, 30);
```

Comment:
I think ideally I'd see some comment why this epoch is why it is.

I know the answer is 'because Excel'.

But we could probably say something like:

> We choose this epoch because 1st of March 1900 should come to be 60 and this epoch gives us that. 
> For days between 1900/01/01 and 1900/02/28 we offset by -1 because the value 60 points to 1900/02/29 which does not really exist.
> Again, value 0 is `1900/01/00` which does not exist again, so for dates prior to 1900/01/01 we need to offset by -2 to reconcile with Excel.

In general - you had to do a deep research on Excel date handling and I think it could be valuable if we could have at least a short summary of the thing (or a pointer to some other docs), so that we can keep a reference of what the offsets are meant to be. IMO such high-level overview of what this converter does would make it easier to understand if it's implemented correctly and maintain it.